src/service/document-info-impl.js

```diff
@@ -172,36 +171,14 @@ function getLinkRels(doc) {
 }
 
 /**
- * Returns a map object of meta tags in document head.
- * Key is the meta name, value is a list of corresponding content values.
+ * Returns the viewport of the document. Note that this is the viewport of the
+ * host document for AmpDocShadow instances.
  * @param {!Document} doc
- * @return {!JsonObject<string, string|!Array<string>>}
+ * @return {?string}
  */
-function getMetaTags(doc) {
-  const metaTags = map();
-  if (doc.head) {
-    const metas = doc.head.querySelectorAll('meta[name]');
-    for (let i = 0; i < metas.length; i++) {
-      const meta = metas[i];
-      const content = meta.getAttribute('content');
-      const name = meta.getAttribute('name');
-      if (!name || !content) {
-        continue;
-      }
-
-      let value = metaTags[name];
-      if (value) {
-        // Change to array if more than one content for the same name
-        if (!isArray(value)) {
-          value = metaTags[name] = [value];
-        }
-        value.push(content);
-      } else {
-        metaTags[name] = content;
-      }
-    }
-  }
-  return metaTags;
+function getViewport(doc) {
+  const viewportEl = doc.head.querySelector('meta[name="viewport"]');
```

Comment:
Why is this being specialized to viewport metas?

Reply:
It's the only `meta` that's currently being used.